ingestion/src/metadata/ingestion/source/database/oracle/metadata.py

```diff
@@ -131,3 +158,107 @@ def get_view_definition(
             logger.debug(traceback.format_exc())
             logger.warning(f"Failed to fetch view definition for {table_name}: {exc}")
         return None
+
+    def process_result(self, data):
+        """Process data as per our stored procedure format"""
+        result_dict = {}
+
+        for row in data:
+            owner, name, line, text = row
+            key = (owner, name)
+            if key not in result_dict:
+                result_dict[key] = {"lines": [], "text": ""}
+            result_dict[key]["lines"].append(line)
+            result_dict[key]["text"] += text
+
+        # Print the concatenated text for each procedure name, ordered by line
+        return result_dict
+
+    def get_stored_procedures(self) -> Iterable[OracleStoredProcedure]:
+        """List Oracle Stored Procedures"""
+        if self.source_config.includeStoredProcedures:
+            results = self.engine.execute(
+                ORACLE_GET_STORED_PROCEDURES.format(
+                    schema=self.context.database_schema.upper()
+                )
+            ).all()
+            results = self.process_result(data=results)
+            for row in results.items():
+                stored_procedure = OracleStoredProcedure(
+                    name=row[0][1], definition=row[1]["text"], owner=row[0][0]
+                )
+                yield stored_procedure
+
+    def get_owner_entity_reference(self, owner_name: str) -> Optional[EntityReference]:
+        """
+        Returns owner's entity reference
+        """
+        owner = None
+        user_owner_fqn = fqn.build(
+            self.metadata, entity_type=User, user_name=owner_name
+        )
+        if user_owner_fqn:
+            owner = self.metadata.get_entity_reference(entity=User, fqn=user_owner_fqn)
+        else:
```

Comment:
as we are returning an owner at the end and as it is under if else block, to avoid two conditions, let's return the owner here itself.
same goes for team owner fqn, if we don't find any fqn, we show warning